### DISCUSSION TITLE
Missing base item tab

### DISCUSSION THREAD

**Discussion by Tibbedoh** [SnosMe/awakened-poe-trade#1193] [Q&A]
All of a sudden I do not have the base item tab anymore.
Updated poe-trade, restarted the game, nothing helps.
Am I missing something (except the tab)?
Update: on some items it does show, but I do not see why. 2 items of the same class, works for one does not for the other.

![image](https://github.com/SnosMe/awakened-poe-trade/assets/71257315/3af8b1a7-7606-4258-9599-81d1cd7d68ac)

**Comment by SnosMe** [comment]
most likely the item has 20% quality, this is one of cases when the item is considered "finished" instead of "raw drop useful for crafting"

**Reply by Tibbedoh** [reply]
Nope, I have 2 axes, same base, both 20%, both fractured, one shows the tabs, the other does not.

**Reply by SnosMe** [reply]
#1095 for futher discussion
